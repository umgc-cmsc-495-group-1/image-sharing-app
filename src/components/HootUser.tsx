import React from 'react';
import { Typography } from '@mui/material';
import { useParams, Outlet } from 'react-router-dom';

export default function HootUser() {
  const { uid } = useParams();
  return (
    <>
      <Typography>
        This is user {uid} profile
      </Typography>
      <Outlet />
    </>
  );
}
