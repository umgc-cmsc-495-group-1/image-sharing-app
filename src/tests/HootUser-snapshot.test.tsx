import React from 'react';
import "@testing-library/react/dont-cleanup-after-each";
import '@testing-library/jest-dom';
import {
  MemoryRouter,
  Routes,
  Route
} from 'react-router-dom';
import HootUser from '../components/HootUser';
import { create } from 'react-test-renderer';

it('Renders correctly', () => {
  const renderer = create(
    <MemoryRouter initialEntries={['/user']}>
      <Routes>
        <Route path='/user' element={<HootUser />} />
      </Routes>
    </MemoryRouter>
  );
  expect(renderer.toJSON()).toMatchSnapshot();
});