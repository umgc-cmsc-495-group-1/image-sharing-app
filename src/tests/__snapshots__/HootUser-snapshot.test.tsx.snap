// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly 1`] = `
<p
  className="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
>
  This is user 
   profile
</p>
`;
